exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/Foreman.java

```diff
@@ -1221,8 +1220,6 @@ public void interrupted(final InterruptedException e) {
      *   to the user
      */
     public void moveToState(final QueryState newState, final Exception ex) {
```

Comment:
With this change, compared to Foreman#moveToState, this listener provides no additional functionality. Please remove this class, and update the docs for Foreman#moveToState accordingly.